rocky/katalogus/views/plugin_detail.py

```diff
@@ -149,20 +163,16 @@ def get_form_filtered_consumable_oois(self):
         oois = self.get_form_consumable_oois()
         return [ooi for ooi in oois if ooi.scan_profile.level >= self.plugin.scan_level.value]
 
-    def get_oois_with_clearance_level(self, selected_oois):
-        """Return a list of selected oois that meets clearance level."""
-        oois_with_clearance_level = []
+    def get_sorted_oois(self, selected_oois: List[str]) -> Dict[str, Any]:
```

Comment:
Naming this `get_sorted_oois` is confusing because this doesn't return a list of sorted OOIs, but a dict with two lists specifying which have clearance level and which haven't.